tests/Doctrine/Tests/ORM/Functional/Ticket/DDC6412Test.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Persistence\Mapping\RuntimeReflectionService;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\Tests\Models\DDC6412\DDC6412File;
+use Doctrine\Tests\OrmTestCase;
+
+/**
+ * @author JarJak
+ */
+class DDC6412Test extends OrmTestCase
```

Comment:
I'd say it's better to leave this check in `ClassMetadataTest.php` only, then we keep it together with the other mapping assertions